### ISSUE TITLE
Dev Test: Ohm's Law 1.4.0-phetiodev.1

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#18]
http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-phetiodev.1/wrappers/index/

This sim was just instrumented and went through a large code refactoring during the code review. The goal is to be able to use this with accessibility sonification wrapper work. It has not been phet brand dev tested, but you can access the plain sim in the above index as well. Assigning to @ariel-phet to prioritize and assign out.

**Comment by zepumph** [comment]
We should probably hammer out https://github.com/phetsims/QA/issues/19 before this.

**Comment by zepumph** [comment]
Substantial work is being done in phetsims/ohms-law#62, and warrants a new dev version, also phetsims/ohms-law#64 (failed to launch) was fixed in master just after this dev release was made. I'm going to close this and make a new issue when things are ready. Thanks @phet-steele for taking a first stab at this sim.
